client/src/components/map/controls/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { PropsWithChildren } from "react";
+import { ChevronDown } from "lucide-react";
+import GlobeSvg from "@/svgs/globe.svg";
+
+import { Collapsible, CollapsibleContent, CollapsibleTrigger } from "@/components/ui/collapsible";
+
+const MapControls = ({ children }: PropsWithChildren) => {
+  return (
+    <Collapsible className="relative space-y-4 rounded-[20px] px-3 py-2 text-sm transition-colors duration-300 hover:bg-background data-[state=open]:bg-background data-[state=open]:pb-3">
+      <CollapsibleTrigger className="group flex items-center gap-2">
```

Comment:
This button has a specific focus style: https://www.figma.com/design/HBpMppIFHR1vA0SbnUJGfO/GRASS---Design-%5BInternal%5D?node-id=499-2947&t=986E6GmpDtbfvv5L-4